docs/modules/ROOT/pages/Java_Builtin_Commands.adoc

```diff
@@ -0,0 +1,104 @@
+= Built-in Commands
+
+Mill comes with a number of useful commands out of the box. These are listed
+in the Scaladoc:
+
+* {mill-doc-url}/api/latest/mill/main/MainModule.html[mill.main.MainModule]
+
+Mill's built-in commands are typically not directly related to building your
+application code, but instead are utilities that help you understand and work
+with your Mill build.
+
+include::example/basicjava/4-builtin-commands.adoc[]
+
+== init
+
+[source,bash]
+----
+> mill -i init com-lihaoyi/mill-scala-hello.g8
+....
+A minimal Scala project.
+
+name [Scala Seed Project]: hello
+
+Template applied in ./hello
+----
```

Comment:
We should rather provide a minimal Java skeleton project.

Reply:
I'm going to remove the `init` portion of this doc for now I think. I'd like to overhaul it at some point later to make it accept the various example projects as templates, but that can come in a follow up PR